diff --git a/node_modules/@xwiki/cristal-lib/dist/index.es.js b/node_modules/@xwiki/cristal-lib/dist/index.es.js
index e6b3761..ee364ac 100644
--- a/node_modules/@xwiki/cristal-lib/dist/index.es.js
+++ b/node_modules/@xwiki/cristal-lib/dist/index.es.js
@@ -7,7 +7,7 @@ import { name as S } from "@xwiki/cristal-document-api";
 import { injectable as $, inject as m, multiInject as A } from "inversify";
 import { createPinia as x } from "pinia";
 import { createI18n as M } from "vue-i18n";
-import { createRouter as W, createWebHashHistory as T } from "vue-router";
+import { createRouter as W, createWebHistory as T } from "vue-router";
 import { CristalLoader as L } from "@xwiki/cristal-extension-manager";
 import { ConfigurationsSettings as R, ComponentInit as j } from "@xwiki/cristal-settings-configurations";
 import { ComponentInit as D } from "@xwiki/cristal-alerts-default";
@@ -274,7 +274,7 @@ let l = class {
        anything special server-side (which is good for partability), and works
        well with electron.
       */
-      history: T(),
+      history: T("/xwiki/cristal"),
       routes: e
     }), this.app = v(Ct).use(this.router).use(x()).use(M({ legacy: !1, fallbackLocale: "en" })), this.app.provide("count", 0), this.app.provide("skinManager", this.skinManager), this.app.provide("cristal", this), this.skinManager.loadDesignSystem(this.app, this.container);
     const n = this.container.getAll(
diff --git a/node_modules/@xwiki/cristal-lib/src/components/DefaultCristalApp.ts b/node_modules/@xwiki/cristal-lib/src/components/DefaultCristalApp.ts
index d802f7d..2d3b6cc 100644
--- a/node_modules/@xwiki/cristal-lib/src/components/DefaultCristalApp.ts
+++ b/node_modules/@xwiki/cristal-lib/src/components/DefaultCristalApp.ts
@@ -33,7 +33,7 @@ import {
   RouteRecordRaw,
   Router,
   createRouter,
-  createWebHashHistory,
+  createWebHistory,
 } from "vue-router";
 import type {
   CristalApp,
@@ -446,7 +446,7 @@ export class DefaultCristalApp implements CristalApp {
        anything special server-side (which is good for partability), and works
        well with electron.
       */
-      history: createWebHashHistory(),
+      history: createWebHistory("/xwiki/cristal"),
       routes,
     });
 
