diff --git a/node_modules/@xwiki/cristal-lib/dist/index.es.js b/node_modules/@xwiki/cristal-lib/dist/index.es.js
index e6b3761..ee364ac 100644
--- a/node_modules/@xwiki/cristal-lib/dist/index.es.js
+++ b/node_modules/@xwiki/cristal-lib/dist/index.es.js
@@ -7,7 +7,7 @@
 import { injectable as $, inject as m, multiInject as A } from "inversify";
 import { createPinia as M } from "pinia";
 import { createI18n as x } from "vue-i18n";
-import { createRouter as T, createWebHashHistory as R } from "vue-router";
+import { createRouter as T, createWebHistory as R } from "vue-router";
 import { CristalLoader as W } from "@xwiki/cristal-extension-manager";
 import { ConfigurationsSettings as L, ComponentInit as F } from "@xwiki/cristal-settings-configurations";
 import { ComponentInit as j } from "@xwiki/cristal-alerts-default";
@@ -321,7 +321,7 @@
       anything special server-side (which is good for partability), and works
       well with electron.
      */
-     history: R(),
+     history: R("/xwiki/cristal"),
      routes: t
    });
    return this.logger?.debug("Replacing state in history " + this.getCurrentPage()), history.replaceState(